### PR TITLE
added link to suggest category on createworkshopform

### PR DESCRIPTION
**Pull request by cinchenx**
Added a link on the CreateWorkshopForm to redirect users to "contact us" if they want to submit a new category

### CODE REVIEW

**Review by goyadev — COMMENTED** [review]
I think there is a conflict that will need to be resolved first

**Comment by cinchenx** [comment]
Resolved the conflict

File: src/components/CreateWorkshopForm.jsx [hunk 1]
```diff
@@ -4,8 +4,8 @@ import Button from "./Button";
 import SuccessfulCard from "./SuccessfulCard";
 import Dropdown from "./Dropdown";
 import CameraIcon from "../assets/icons/camera.svg";
-import postWorkshop from "../api/post-workshop";
```
Comment (goyadev) [inline src/components/CreateWorkshopForm.jsx:7]: I could be wrong, but I think we might need post Workshop and the loadspinner still?

Comment (suanTech) [reply src/components/CreateWorkshopForm.jsx:7]: Yep we dooo!
If Cindy can ctrl+z to revert it back, and push again that'll be great!!

Comment (cinchenx) [reply src/components/CreateWorkshopForm.jsx:7]: I keep getting an error when I try to push?  my steps are: 
git pull origin development 
modified createworkshopform.jsx 
git add file 
git commit 
git push.
![Screen Shot 2024-04-05 at 2 13 38 PM]

**Comment by cinchenx** [comment]
Screen Shot 2024-04-05 at 2 15 38 PM

**Review by goyadev — APPROVED** [review]
test

**Pull request closed without merging**